HaskellNet.cabal

```
@@ -67,14 +67,14 @@ Library
     Network.Mail.Mime
 
   Build-Depends:
-    base >= 4.3 && < 4.19,
+    base >= 4.3 && < 4.20,
     network >= 2.6.3.1 && < 3.2,
     mtl >= 2.2.2 && < 2.4,
-    bytestring >=0.10.2 && < 0.12,
+    bytestring >=0.10.2 && < 0.13,
     pretty >= 1.1.3 && < 1.2,
     array >= 0.5 && < 0.6,
     cryptohash-md5  >= 0.11 && < 0.12,
-    base64 < 0.5,
+    base64 < 1.1,
```

Maybe this should be:

```suggestion
    base64 >= 1.0 && < 1.1,
```